spacewalk/setup/bin/spacewalk-setup

```diff
@@ -1163,8 +1163,24 @@ sub setup_services {
   return 1;
 }
 
+sub is_suse_like {
+  my $line;
+  if(! -e '/etc/os-release') {
+    return undef;
+  }
+  open(OS_RELEASE,"/etc/os-release") || die ("Could not open /etc/os-release\n");
```

Comment:
`os-release` is a file that comes with `systemd`.
What about distributions that still relies on old `SysVinit` (e.g. SLE11)?

Reply:
You'll have to do both. Just check for SuSE-release first and then for os-release.

Reply:
This only for the server, and server only accepts SLE15/openSUSE Leap15.

Other distributions will be considered as non SUSE-Like, and will use the other part of the code.